docs/pywbemclicommands.rst

```diff
@@ -849,7 +849,7 @@ used to create a connection and is largely the same information as is in the
 options for pywbemcli. The data includes:
 
 * **name** name of the connection (required).
-* **server_url** the url for the defined connection (required unless
+* **server** the url for the defined connection (required unless
   ``--mock-server``/``-m`` defined).
 * **default_namespace** the default namespace defined for the connection
```

Comment:
The names of the data items are not in consistent with the implementation w.r.t. hyphens vs underscores. 

I suggest to always use underscores for property names in YAML (or JSON), and not hyphens. That makes creating dicts in Python with such properties simpler, for example because dicts can be initialized with the property names as named arguments, e.g. dict(connection_definitions=dict()).